samples/snippets/subscriber.py

```diff
@@ -580,6 +580,55 @@ def callback(message: pubsub_v1.subscriber.message.Message) -> None:
     # [END pubsub_subscriber_blocking_shutdown]
 
 
+def receive_messages_with_exactly_once_subscribe(
+    project_id: str, subscription_id: str, timeout: Optional[float] = None
+) -> None:
+    """Receives messages from a pull subscription with exactly-once delivery enabled."""
+    # [START pubsub_subscriber_pull_with_exactly_once_delivery_enabled]
```

Comment:
Please change this to `pubsub_subscriber_exactly_once` (as described in b/221257565)

Reply:
pubsub_subscriber_exactly_once_delivery please.